near-sdk/src/collections/legacy_tree_map.rs

```diff
@@ -0,0 +1,1769 @@
+//! Legacy `TreeMap` implementation that is using `UnorderedMap`.
+//! DEPRECATED. This implementation is deprecated and may be removed in the future.
```

Comment:
Hmm, but what it the point of having such implementation then?

Reply:
Because some contracts might still be using it.